26_map-filter-reduce/26_map-filter-reduce.cljs

```diff
@@ -0,0 +1,30 @@
+;; -=-=--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-==--=
+;; An Introduction to ClojureScript for Light Table users
+;; -==-=--=-=-=-=-=-=-=-=-==-=--=-=-==--==-=-=-=-=--===--==-
+
+(ns lt-cljs-tutorial
+  (:require [clojure.string :as string]))
+
+
+;; Map / Filter  / Reduce
+;;=----=----------------=
+
+
+;; ClojureScript supports the same bells and whistles out of the box that you may
+;; be familiar with from other functional programming language or JavaScript
```

Comment:
+;; be familiar with from other functional programming ~~language~~ languages or JavaScript